addons/mail/static/src/messaging/entity/message/message.js

```diff
@@ -188,6 +188,20 @@ function MessageFactory({ Entity }) {
             });
         }
 
+        /**
+         * Mark provided messages as read.
+         *
+         * @static
+         * @param {mail.messaging.entity.Message[]} messages
+         */
+        static async markAsRead(messages) {
+            await this.env.rpc({
```

Comment:
this.async

Reply:
Or not because it's static...

Return is by longpolling btw?

Reply:
Yes, will add a comment for that.